builtin/v16/evm/evm_state.go

```diff
@@ -18,6 +28,7 @@ type State struct {
 	Bytecode      cid.Cid
 	BytecodeHash  [32]byte
 	ContractState cid.Cid
+	TransientData *TransientData
```

Comment:
```suggestion
    // The data that containers tranisent state. The struct includes the CID of the KAMT that
    // contains the state data and the tuple of nonce and actor id that represents
    // the lifespan of the transient storage data
	TransientData *TransientData
```

I know this struct lacks godocs but while you're in here would you mind doing a quick pass to copy across at least some basic docs for it all from actors/evm/src/state.rs?

Reply:
e709224 updated docs!